definitions/procedures/backup/snapshot/mount_pulp.rb

```diff
@@ -0,0 +1,24 @@
+require 'procedures/backup/snapshot/mount_base'
+module Procedures::Backup
+  module Snapshot
+    class MountPulp < MountBase
+      metadata do
+        description 'Create and mount snapshot of Pulp data'
+        tags :backup
+        for_feature :pulp
+        MountBase.common_params(self)
+        param :skip, 'Skip Pulp content during backup'
```

Comment:
I wonder, if we shouldn't use the `--whitelist` functionlity for skipping?

Reply:
Nm: I see the context mapping you're using, probably not worth the effort to change that.